modules/platforms/python/cpp_module/type_conversion.h

```diff
@@ -185,6 +377,10 @@ static void submit_pyobject(ignite::binary_tuple_builder &builder, PyObject *obj
 
     if (PyLong_Check(obj)) {
         auto val = PyLong_AsLongLong(obj);
+        if (PyErr_Occurred()) {
+            throw ignite::ignite_error("Can not convert INT to a long long");
```

Comment:
Same as above - is that possible in practice?

Reply:
This is actually possible, when a really big int is used in Python (e.g., I tried `423538409739423538409739423538409739423538409739423538409739423538409739423538409739` and got this error)